components/02-elements/form/radio/_radio.scss

```diff
@@ -12,10 +13,28 @@ $radio__icon-transition   : $transition-base !default;
     position: relative;
     margin-bottom: $radio__margin;
 
+    &__fieldset {
+        border: 0;
+        padding: 0;
+        margin: 0;
+    }
+
+    &__legend {
+        padding-bottom: $radio__vertical-gap;
+    }
+
+    &__handler {
+        position: relative;
+    }
+
     &__field {
-        display: none;
+        position: absolute;
+        left: 0;
+        top: $radio__vertical-gap;
```

Comment:
why it should be 10px? If we change to 0 the focus of screen readers it's better. Could we change it or there are some hidden feature of this? BTW, let's add `!default` to this variable :)